ctranslate2/run_whisper.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+export PYTHONPATH="..":$PYTHONPATH
+
+MODEL_IDs=("guillaumekln/faster-whisper-tiny.en" "guillaumekln/faster-whisper-small.en" "guillaumekln/faster-whisper-base.en" "guillaumekln/faster-whisper-medium.en" "guillaumekln/faster-whisper-large-v1" "guillaumekln/faster-whisper-large-v2")
+BATCH_SIZE=1
+DEVICE_INDEX=0
+
+num_models=${#MODEL_IDs[@]}
+
+for (( i=0; i<${num_models}; i++ ));
+do
+    MODEL_ID=${MODEL_IDs[$i]}
+
+    python run_eval.py \
+        --model_id=${MODEL_ID} \
+        --dataset_path="open-asr-leaderboard/datasets-test-only" \
```

Comment:
Can you update the paths here to be: https://huggingface.co/datasets/hf-audio/esb-datasets-test-only